src/libraries/System.Formats.Asn1/ref/System.Formats.Asn1.cs

```diff
@@ -179,7 +179,7 @@ public sealed partial class AsnWriter
         public void WriteNull(System.Formats.Asn1.Asn1Tag? tag = default(System.Formats.Asn1.Asn1Tag?)) { }
         public void WriteObjectIdentifier(System.ReadOnlySpan<char> oidValue, System.Formats.Asn1.Asn1Tag? tag = default(System.Formats.Asn1.Asn1Tag?)) { }
```

Comment:
Nit: should oidvalue also be renamed to value as well?

Reply:
`oidValue` was actually intentional, to help convey that from the `Oid` type you want to pass `oid.Value`, and that `oid.FriendlyName` is not supported.

Good question, though.